packages/launchpad/src/setup/EnvironmentSetup.vue

```diff
@@ -1,12 +1,52 @@
 <template>
+  <Alert
+    v-model="isAlertOpen"
+    :icon="ErrorOutlineIcon"
+    class="mx-auto my-24px max-w-640px"
+    status="warning"
+    :title="t('setupPage.projectSetup.communityFrameworkDefinitionProblem')"
+    dismissible
+  >
+    <p>
+      {{ t('setupPage.projectSetup.communityDependenciesCouldNotBeParsed', erroredFrameworks.length) }}
+    </p>
+    <ul class="list-disc my-12px ml-36px">
+      <li
+        v-for="framework in erroredFrameworks"
+        :key="framework.path as string"
+      >
+        <ExternalLink
+          data-cy="errored-framework-path"
+          :href="`file://${framework.path}`"
+        >
+          {{ framework.path }}
+        </ExternalLink>
+      </li>
+    </ul>
+    <i18n-t
+      tag="p"
+      keypath="setupPage.projectSetup.seeFrameworkDefinitionDocumentation"
+    >
+      <ExternalLink
+        :href="getUrlWithParams({
+          url :'https://on.cypress.io/component-integrations',
+          params: {
+            utm_medium: 'Framework Definition Warning'
```

Comment:
I don't see this in the UTM params spreadsheet

Reply:
Good point, and it should be tested.

Reply:
Added to sheet

Reply:
Updated the component test to check the param.